lib/subscribers/langchain/runnable-stream.js

```diff
@@ -0,0 +1,151 @@
+/*
+ * Copyright 2025 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+const LangchainSubscriber = require('./base')
+const { AI: { LANGCHAIN, STREAMING_DISABLED } } = require('../../metrics/names')
+
+class LangchainRunnableStreamSubscriber extends LangchainSubscriber {
+  constructor({ agent, logger }) {
+    super({ agent, logger, channelName: 'nr_stream' })
+    this.events = ['asyncEnd']
+  }
+
+  get enabled() {
+    return super.enabled && this?.agent?.config?.ai_monitoring?.streaming?.enabled
```

Comment:
No need to do optional chaining

```suggestion
    return super.enabled && this.agent.config.ai_monitoring.streaming.enabled
```